crates/nu-protocol/src/ast/expr.rs

```diff
@@ -42,7 +45,8 @@ pub enum Expr {
     ImportPattern(Box<ImportPattern>),
     Overlay(Option<BlockId>), // block ID of the overlay's origin module
     Signature(Box<Signature>),
-    StringInterpolation(Vec<Expression>),
+    /// The boolean is `true` if the string is quoted.
+    StringInterpolation(Vec<Expression>, bool),
```

Comment:
Isn't string interpolation always quoted? When is string interpolation _not_ quoted? I can think only of bareword interpolation.

Also a code style note: We should ban the use of booleans in places like that, this should be signalized by an enum. Not worth fixing atm, but something to keep in mind for the new parser.

Reply:
Yes, just bareword interpolation.

Does 
```nushell
$`string`
```
syntax currently exist? If so, I wonder how that should be represented. Backtick globs are expanded I think

Reply:
No, this currently isn't implemented, though I think we agreed at some point that it's worth adding.